.travis.yml

```diff
@@ -12,49 +12,18 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      services:
-        - mysql
-      env:
-        - DEPS=latest
-        - TEST_INTEGRATION=true
-        - TESTS_LAMINAS_DB_ADAPTER_DRIVER_MYSQL=true
-    - php: 7.0
-      env:
-        - DEPS=lowest
-    - php: 7.0
-      services:
-        - mysql
-      env:
-        - DEPS=latest
-        - TEST_INTEGRATION=true
-        - TESTS_LAMINAS_DB_ADAPTER_DRIVER_MYSQL=true
-    - php: 7.1
-      env:
-        - DEPS=lowest
-    - php: 7.1
-      services:
-        - mysql
-      env:
-        - DEPS=latest
-        - CS_CHECK=true
-        - TEST_COVERAGE=true
```

Comment:
We need to move these to run CS CHECK and TEST COVERAGE on one of the supported PHP versions.

Reply:
I `did` that previously, but need to revert it based on @weierophinney comment at https://github.com/laminas/laminas-db/pull/164#pullrequestreview-424476322

Reply:
applied